### DISCUSSION TITLE
Moving sbt-sonatype to sbt orgnization

### DISCUSSION THREAD

**Discussion by xerial** [xerial/sbt-sonatype#502] [Ideas]
As sbt-sonatype has become a part of the Scala ecosystem (e.g., used for releasing Scala 3), acquiring more maintainers (e.g., scala center) and enabling direct stakeholders to fix the plugin is becoming critical. 

Let's explore potential directions in this thread.

Current challenges lie in various domains and user authentication systems:
- Sonatype legacy host (oss.sonatype.org)
- New sonatype host (s01.oss.sonatype.org)
- The new Sonatype Central user authentication system, which is separated from the legacy host. 
  -  Different APIs need to be used for publishing OSS artifacts

**Comment by eed3si9n** [comment]
If you want to move the repo to sbt org, please let me know. I can set up a team with whoever you want in it.
